packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -46,6 +41,27 @@ if ([production, development].indexOf(mode) < 0) {
     throw new Error(`Invalid mode "${mode}"`)
 }
 
+// for API convenience, add the leading slash if missing
+export const EXT_OVERRIDES_DIR =
+    typeof pkg?.ccExtensibility?.overridesDir === 'string' &&
+    !pkg?.ccExtensibility?.overridesDir?.startsWith('/')
+        ? '/' + pkg?.ccExtensibility?.overridesDir
+        : pkg?.ccExtensibility?.overridesDir
```

Comment:
Same, I think is good we implement a fail-safe logic instead of a fail-fast solution, but at least I think we need to inform the user we are ignoring the invalid value defined and we proceed with our safe defaults.

Reply:
I actually think that we want to say "both are valid" so in that sense, not warn the user